src/zui/ZUIDataTableSearch.tsx

```diff
@@ -1,14 +1,11 @@
-import { makeStyles } from '@mui/styles';
 import {
-  Badge,
   Box,
-  Button,
   Fade,
   IconButton,
-  Popover,
   TextField,
 } from '@mui/material';
-import { Close, Search } from '@mui/icons-material';
+import { Close } from '@mui/icons-material';
+import SearchIcon from '@mui/icons-material/Search';
```

Comment:
Is there a specific reason to change this import?

Reply:
no reason, removed now